docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2039 @@
+{
```

Comment:
<p>Perhaps mention after this block that Model also has methods to extract the gradient (dlogp) and hessian (d2logp) of the logp and link to the model entry in the docs?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='113'/>

Reply:
<p>+1</p>

Reply:
+1